### PR TITLE
Av 75987 Added support to include the role

### PR DESCRIPTION
**Pull request by Rohan-sss1**

### CODE REVIEW

**Comment by grastogi23** [comment]
Need tests

**Comment by Rohan-sss1** [comment]
Adding test cases in avi-dev to test the role on Jenkins

File: tasks/main.yml [hunk 1]
```diff
@@ -7,3 +7,36 @@
     msg="This role works best with ansible version 2.4 or greater.
     It is recommended to use the latest ansible version for
     the best experience."
+
+- name: Avi Controller | Vmware | Check Requirements
+  include: requirements.yml
+
+- name: Avi Vmware | Deploy controller
```
Comment (grastogi23) [inline tasks/main.yml]: shouldn't this be setup with version?

Comment (Rohan-sss1) [reply tasks/main.yml]: yes, we removed version.

**Comment by Rohan-sss1** [comment]
Test cases for this role.
https://github.com/avinetworks/avi-dev/pull/62055

**Pull request merged**